channeldb/migration/lnwire21/msat.go

```diff
@@ -55,8 +55,10 @@ func (m MilliSatoshi) String() string {
 // Record returns a TLV record that can be used to encode/decode a MilliSatoshi
 // to/from a TLV stream.
 func (m *MilliSatoshi) Record() tlv.Record {
+	msat := uint64(*m)
+
 	return tlv.MakeDynamicRecord(
-		0, m, tlv.SizeBigSize(m), encodeMilliSatoshis,
+		0, m, tlv.SizeBigSize(&msat), encodeMilliSatoshis,
```

Comment:
Do you realize that this is old migration code?

Reply:
I think what Laolu means is: The whole package `migration/lnwire21` is only used by the migration with the number 21, which is ancient. So in practice it will never run again on any existing node.

Reply:
migration32 imports this package and uses the types there - and migration32 has not yet been included in a release - so i think this will be used there no?

Reply:
Ah, yes, now I see it... I searched for the wrong package.